plugins/node/opentelemetry-instrumentation-ioredis/src/utils.ts

```diff
@@ -34,7 +75,17 @@ export const endSpan = (
 export const defaultDbStatementSerializer: DbStatementSerializer = (
   cmdName,
   cmdArgs
-) =>
-  Array.isArray(cmdArgs) && cmdArgs.length
-    ? `${cmdName} ${cmdArgs.join(' ')}`
-    : cmdName;
+) => {
+  if (Array.isArray(cmdArgs) && cmdArgs.length) {
+    const argsSubset = SerializationSubsets[cmdName.toUpperCase()];
+    if (argsSubset) {
+      const args = cmdArgs.slice(0, argsSubset);
+      const remainder = cmdArgs.slice(argsSubset);
+      return `${cmdName} ${args.join(' ')} [${
+        remainder.length
+      } other arguments]`;
+    }
+    return `${cmdName} ${cmdArgs.join(' ')}`;
```

Comment:
We must not fallback on joining every args in the result, could you change this to `cmdArgs[0]` ?